src/globus_sdk/_testing/registry.py

```diff
@@ -53,6 +58,16 @@ def _resolve_qualname(name: str) -> str:
 
     assert issubclass(maybe_client, globus_sdk.BaseClient)
     service_name = maybe_client.service_name
+
+    # TODO: Consider alternative strategies for mapping versioned clients
+    # to subdirs. For now, we do it by name matching.
```

Comment:
Hah! I think that instinct comes from a good place -- there are some folks who use "TODO" as an excuse to do shoddy work and punt hard problems into the future, and if you at least have to describe the problem, it means you've thought about it.

In this case, this was a solution Reid and I arrived at in a discussion which we think is... fine. But not necessarily perfect or great. It notably means that response loading by method name lookups will differentiate between `ComputeClient` and `ComputeClientV2`. The big alternative we played with was to add a client attribute which declares the "fixture subdir" or "api version (but only semantically useful for finding the SDK fixture subdir)". I didn't love that approach either, on account of how ultra-specific it would be to this use-case. We debated, and I wanted to capture our uncertainty about the solution being the long-term best option by calling out that we should think about this more in the future.